tools/local-presubmit

```diff
@@ -11,8 +11,10 @@ source $(dirname $0)/logging.sh
 cd $ROOT
 status "Testing all of the things."
 
+set -x
+
 ./tools/sigh webpack
-./tools/bazelisk test //java/... //javatests/... //src/... //particles/...
+./tools/bazelisk --bazelrc=tools/travis.bazelrc test //java/... //javatests/... //src/... //particles/...
```

Comment:
This script is intended to be used as a "test everything" script, for local development. I'm not sure if many people use it for that though... The `set -x` is probably ok to keep, but I worry about using the travisrc file locally. It might include options that would invalidate the local bazel cache.

Can you think of any way to reuse this script on Cloud Build without hardcoding this flag here? Maybe a separate `tools/cloud-build-presubmit` script that does something like this:

```
cp tools/travis.bazelrc .bazelrc
tools/local-presubmit
```

Or you could add an optional bazelrc flag to this script maybe?

Reply:
I added a flag to tools/local-presubmit. It can be removed once we split up the commands in the script into different steps in cloudbuild.yaml.